docs/tutorials/auth-service-migration.md

```diff
@@ -0,0 +1,278 @@
+---
+id: auth-service-migration
+title: Migrating to New Auth Services
+description: A guide for how to migrate Backstage backend plugins and modules to use the new auth services
+---
+
+The auth services for the Backstage backend system have been reworked in the 1.24 release. Among other improvements, this brings protection by default for Backstage backends, replacing the [authenticate-api-requests.md](https://github.com/backstage/backstage/blob/9aac2b0d36bdb8095ea747fe5e5490cfea1c9f16/contrib/docs/tutorials/authenticate-api-requests.md) guide in contrib. This guide will help you migrate your existing backend setup as well as backend _plugins and modules_ to use the new auth services.
+
+The change with the most impact that accompany the new auth services is the default behavior of all plugins running in the new backend system to block all requests that are not authenticated as a user or service, also known as the _default auth policy_. This is the only breaking production change introduces as part of this update, and may require action from both backend installations and plugins. More on that in the individual sections below.
+
+## Backend migration
+
+In order to use these new services your backend needs to be using the [new backend system](../backend-system/building-backends/01-index.md). If your backend is running the old system, you will need to [migrate it to the new system first](../backend-system/building-backends/08-migrating.md).
+
+If you have [authenticate-api-requests.md](https://github.com/backstage/backstage/blob/9aac2b0d36bdb8095ea747fe5e5490cfea1c9f16/contrib/docs/tutorials/authenticate-api-requests.md) installed in your backend, you should generally remove it and rely on the new auth services instead. If you do not wish to make that change yet but still want to upgrade to the latest release of Backstage, you can also leave it in place and instead disable the default auth policy as described in the next section.
+
+### Disabling the default auth policy
+
+If you do not want to enforce authentication of requests by default, you can disable the default auth policy. This is done using the following configuration:
+
+```yaml
+backend:
+  auth:
+    dangerouslyDisableDefaultAuthPolicy: true
+```
+
+In short, this will allows requests through to plugins in your backend, even if they do not include any credentials. The requests will still be treated as unauthenticated however, which not all plugin endpoints may accept. For more information on the impact of this configuration, see the [auth service documentation](../backend-system/core-services/auth.md).
```

Comment:
```suggestion
In short, this will allow requests through to plugins in your backend, even if they do not include any credentials. The requests will still be treated as unauthenticated however, which not all plugin endpoints may accept. For more information on the impact of this configuration, see the [auth service documentation](../backend-system/core-services/auth.md).
```